### DISCUSSION TITLE
Error when using `local_future_into_py` or local_future_into_py_with_locals` using pyo3-asyncio

### DISCUSSION THREAD

**Discussion by Tom-the-Bomb** [PyO3/pyo3#2561] [Questions] [answered]
Im having some problems with pyo3-asyncio and it's `local_future_into_py` / `local_future_into_py_with_locals` where when I try to run the python functions I get 
```
pyo3_runtime.PanicException: `spawn_local` called from outside of a `task::LocalSet`
```
I am aware that lib is (3rd party)? but since it's related so I thought I might try asking here anyways.

code here: https://github.com/Tom-the-Bomb/akinator.py

thanks in advance

**Comment by asze17** [comment, marked as answer]
I have responded to you in https://github.com/awestlake87/pyo3-asyncio/issues/71#issuecomment-1218478694
